modules/lib/lib-grid/src/test/resources/test/grid-test.js

```diff
@@ -0,0 +1,108 @@
+var gridLib = require('/lib/xp/grid');
+var assert = require('/lib/xp/testing');
+
+exports.testGet_without_key = function () {
+    var memoryGrid = gridLib.getMap('mapId');
+    try {
+        memoryGrid.get({});
+    } catch (e) {
+        assert.assertEquals(`Parameter "key" is required`, e);
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9567561)
```suggestion
        assert.assertEquals('Parameter "key" is required', e);
```